drivers/scheduler/k8s/k8s.go

```diff
@@ -1204,8 +1225,13 @@ func (k *K8s) createStorageObject(spec interface{}, ns *corev1.Namespace, app *s
 	if obj, ok := spec.(*storageapi.StorageClass); ok {
 		obj.Namespace = ns.Name
 
-		logrus.Infof("Setting provisioner of %v to %v", obj.Name, volume.GetStorageProvisioner())
-		obj.Provisioner = volume.GetStorageProvisioner()
+		if volume.GetStorageProvisioner() != PortworxStrict {
+			logrus.Infof("Setting provisioner of %v to %v", obj.Name, volume.GetStorageProvisioner())
+			obj.Provisioner = volume.GetStorageProvisioner()
+		} else {
+			logrus.Infof("Setting provisioner of %v to %v", obj.Name, obj.Provisioner)
+		}
+
```

Comment:
you can remove else here as it is redundant
```
if volume.GetStorageProvisioner() != PortworxStrict {
			obj.Provisioner = volume.GetStorageProvisioner()
} 
logrus.Infof("Setting provisioner of %v to %v", obj.Name, obj.Provisioner)
```

Reply:
Done..